app/models/user.rb

```diff
@@ -0,0 +1,2 @@
+class User < ApplicationRecord
```

Comment:
Since you don't allow email to be null. It would be also good to add validation on model

Reply:
Also I would think if you should allow duplicated emails if not add uniqueness validation and unique index on db